lib/cocoapods-core/specification/dsl.rb

```diff
@@ -463,6 +463,25 @@ module DSL
 
       #------------------#
 
+      # @!method static_framework=(flag)
+      #
+      #   Indicates, that if !use_frameworks is specified, the
+      #   pod should include a static library framework.
+      #
+      #   @example
+      #
+      #     spec.static_framework = true
+      #
+      #   @param [Bool] flag
+      #          Indicates, that if !use_frameworks is specified, the
+      #          pod should include a static library framework.
```

Comment:
what does "include" mean in this case? Does it mean "if use_frameworks! is specified then this pod will be packaged as a static framework?

Also here and throughout `!use_frameworks` should be `use_frameworks!`